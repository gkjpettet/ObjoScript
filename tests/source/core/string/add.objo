System.print("abc" + 1) # Expect abc1
System.print("abc" + true) # Expect abctrue
System.print("abc" + "d") # Expect abcd
System.print("abc" + [1, 2]) # Expect abc[1, 2]
System.print("abc" + {"key" : "value"}) # Expect abcd{key : value}