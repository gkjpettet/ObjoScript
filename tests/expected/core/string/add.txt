abc1
abctrue
abcd
abc[1, 2]
abc{key : value}